lib/Components/Cart/cart.html

```diff
@@ -0,0 +1,306 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SmashCart - Home - Manoj Sarna</title>
+    <link rel="icon" type="image/x-icon" href="/lib/assets/favicon/favicon.ico">
+    <link rel="stylesheet" href="cart.css">
+
+</head>
+
+<body>
+    <header class="sm-header">
+        <nav class="sm-nav-new">
+            <!-- LOGO -->
+            <a href="/" class="link-text-dec" title="Go To Home">
+                <div id="sc-logo" class=" sm-nav-logo flex-row-wrap">
+                    <span class="sm-header-icon">
+                        <i class="fab fa-opencart"></i>
+                    </span>
+                    <span>SmashCart</span>
+                </div>
+            </a>
+
+            <!-- SEARCH -->
+            <div class="sm-nav-search">
+                <div class="sm-input-container input-search">
+                    <input type="text" class="input-basic input-basic-icon" placeholder="Search..." name="search" />
+                    <i class="icon-basic fas fa-search"></i>
+                </div>
+            </div>
+            <!-- ICONS BUTTONS FOR CTA -->
+            <div class="sm-action-icon-btn-wrapper">
+                
+                <!-- Cart Icon -->
+                <button id="cart-btn-header" class="sm-icon-btn color-dm sm-icon-btn-primary sm-action-icon-wishlist">
+                    <div class="icon">
+                        <i class="fas fa-shopping-cart"></i>
+                        <span class="icon-badge bd-red icon-bd-top-right ">10</span>
+                    </div>
+                </button>
+
+                <!-- Wishlist Icon -->                
+                <button id="wishlist-btn-header" class="sm-icon-btn color-dm sm-icon-btn-primary sm-action-icon-cart">
+                    <div class="icon">
+                        <i class="fas fa-heart"></i>
+                        <span class="icon-badge bd-red icon-bd-top-right ">8</span>
+                    </div>
+                </button>
+
+                <!-- DARK MODE BUTTON -->
+                <button id="btn-dm" class="btn-social btn-dark-mode ">
+                    <i class="fas fa-moon" title="Switch To Dark Mode"></i>
+                </button>
+
+                <!-- LOGIN USER -->
+                <button class="btn-social btn-logged-in-user">
+                    <i class="fas fa-user" title="Check Profile , Can Logout!!"></i>
```

Comment:
You used title attribute which is pretty neat. Great work.

Reply:
NEVER DO THIS - +1